pkgs/tools/networking/uwimap/default.nix

```diff
@@ -19,6 +19,7 @@ stdenv.mkDerivation rec {
   hardeningDisable = [ "format" ];
 
   buildInputs = [ openssl ]
+    ++ lib.optional stdenv.isDarwin libkrb5
     ++ lib.optional (!stdenv.isDarwin) pam;
```

Comment:
```suggestion
    ++ lib.optional stdenv.isLinux pam;
```

Reply:
I'm actually going to change this to match the make target condition. Especially if we want to broaden this to `platforms.unix`, because I believe PAM is not Linux exclusive.